cmd/argocd-util/main.go

```diff
@@ -109,7 +109,7 @@ func NewRunDexCommand() *cobra.Command {
 				} else {
 					err = ioutil.WriteFile("/tmp/dex.yaml", dexCfgBytes, 0644)
 					errors.CheckError(err)
-					log.Info(redactor(string(dexCfgBytes)))
+					log.Debug(redactor(string(dexCfgBytes)))
```

Comment:
I agree this is better. Operator still can either enable debugging or just looks at `/tmp/dex.yaml` within the container.